<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorWindowClass</class>
 <widget class="QDialog" name="AuthorWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthorWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stWidg_auth">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_auth">
      <widget class="QLabel" name="lb_name">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_passwd">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEd_name">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEd_passwd">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_regest">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_logIn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_cancel">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_regest">
      <widget class="QLabel" name="lb_regRepPW">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Repeat PW</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEd_repPasWD">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_regCancel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_regPassWD">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_regRegistr">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEd_regRepPW">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnEd_regName">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lb_regName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pB_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AuthorWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
